app/connectors/SubscriptionConnector.scala

```diff
@@ -85,7 +84,18 @@ class SubscriptionConnector @Inject() (val config: FrontendAppConfig, val http:
     }
   }
 
-  def handleError[A](responseMessage: HttpResponse): Either[ApiError, A] =
+  def updateSubscription(requestDetail: UpdateSubscriptionRequest)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
+    val updateSubscriptionUrl = s"${config.businessMatchingUrl}/subscription/update-subscription"
+    http
+      .PUT[UpdateSubscriptionRequest, HttpResponse](updateSubscriptionUrl, requestDetail)
```

Comment:
Looking at the spec in DAC6-3049, this API should be a `POST` API not a `PUT` API. Please can you change it from PUT to POST as part of this work? Save creating / estimating a whole new ticket for a 5 minute job.

Reply:
@jagankumar-hmrc confirmed a while back that it ought to be a `PUT`. I suspect the spec hasn't been updated yet so I've messaged him if he could be kind enough to do so.

Reply:
@jagankumar-hmrc has now updated the spec to a `PUT`.